README.md

```diff
@@ -133,6 +133,36 @@ int main(void){
 }
 ```
 
+## Enabling traces in mbed OS 5
+
+To enable traces in mbed OS 5, add the following to your ``mbed_app.json``:
+
+```json
+{
+   "target_overrides": {
+      "*": {
+         "mbed-trace.enable": 1
+      }
+   }
+}
+```
+
+Then instantiate the tracing module, and forward data to stdout by adding the following to your ``main.cpp``:
+
+```cpp
+#include "mbed-trace/mbed_trace.h"
+
+void trace_printer(const char* str) {
+   printf("%s\r\n", str);
+}
```

Comment:
Providing the trace_printer function is also not strictly speaking necessary if all you want to do is get traces. It's currently used in a lot of apps to make cli prints and traces to play along better.